lunatrace/bsl/frontend/src/pages/account/admin.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import React, { useState } from 'react';
+import { Button, Card, Col, Container, Form, ListGroup, ListGroupItem, Row } from 'react-bootstrap';
+import { useNavigate } from 'react-router-dom';
+
+import { api } from '../../api/generated';
+import { impersonateUserHeader } from '../../constants/headers';
+import { ImpersonateUser } from '../../types/user';
+import { inputChangeHandler } from '../../utils/input-helpers';
+import { setImpersonatedUser } from '../../utils/users';
+
+export const AdminDashboard = () => {
+  const navigate = useNavigate();
+  const [username, setUsername] = useState<string>('');
+
+  const { data, error, isLoading } = api.endpoints.GetLunaTraceUsers.useQuery();
```

Comment:
That's interesting.  They also should be populating hooks for us onto the api object so you can do 
`api.useGetLunaTraceUsersQuery` and it should work. Idk, maybe you doing it this way has to do with the type struggles you were having.